### DISCUSSION TITLE
Not able to detect Vertical surface [Wall]

### DISCUSSION THREAD

**Discussion by jeetendra-kumar** [sceneview/sceneform-android#260] [Q&A] [answered]
I am trying to detect vertical surface [wall] but not able to detect. And when i am going to check setPlaneFindingMode method inside config.java its not saving and not doing anything. Can anyone suggest how i can detect vertical [wall] surface?

**Comment by grassydragon** [comment, marked as answer]
Hi!
Maybe the problem is that the wall doesn't have enough texture to be recognized?
